Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -2,8 +2,7 @@
 
 #pragma once
 
-#include <improbable/c_worker.h>
-#include <improbable/collections.h>
+#include "SchemaOption.h"
```

Comment:
We still need <improbable/c_worker.h> here as we are referencing `Worker_EntityId`